test/integration/kubectl/kubectl_test.go

```diff
@@ -56,6 +56,7 @@ func TestKubectlValidation(t *testing.T) {
 	cfg.CurrentContext = "test"
 	overrides := clientcmd.ConfigOverrides{
 		ClusterInfo: *cluster,
+		Context:     *ctx,
```

Comment:
Hm, I agree, it looks to not be needed, I've tested this locally and the kubectl integration test is passing without it. I've gone ahead and removed it